docs-src/docs/openlabel/openlabel-format.md

```diff
@@ -392,3 +392,63 @@ A 3D lane is represented as two lines in 3D (`poly3d`), one to the right and the
   }
 }
 ```
+
+## Representing 2D points
+
+A 2D point is represented, as you might expect, as a single point2d. The caveat is that we export it with the attributed `point_class`. Due to future representation of other point geometries, this attribute has to be equal to the `type` of the point object. The `point_class` is also completely optional for 2D points so omitting it is treated the same as if the `point_class` was equal to the `type`
```

Comment:
I suggest we're clear about the cases we support and guide people towards exactly those, so, perhaps say something like...

> A 2D point is represented with a single `point2d` geometry. Each `point2d` has an optional `point_class` attribute. For single points this may be omitted, otherwise it must be equal to the `type` of the object. This attribute is reserved for future use on other point-based geometries.

(I'm not sure if `point_class` was _optional_ or if it can just be left blank - I'll leave it to you to clarify that if needed!)

Reply:
I like it so I'll mostly steal it. Do you think it's clear from omitted that you either don't include an attribute with name `point_class` or you include it but with the value equal to `type`.

Reply:
As it is now, it's clear yep!